platform/core.windows/apichanges.xml

```diff
@@ -84,6 +84,26 @@ is the proper place.
 
   <changes>
 
+    <change id="preferred-color-profile-spi" >
+        <api name="winsys"/>
+        <summary>Add preference for sorting editor tabs</summary>
```

Comment:
Wrong summary copy-pasted in?

Reply:
Yes. Copy+paste issue, fixing.

Reply:
:+1: I think this now has preferred spelt wrong though?

Reply:
Yes it was. Thanks!

Reply:
Thanks!  Haven't tested yet, but use of SPI over reflection hacks is a good move.